package.json

```diff
@@ -11,7 +11,8 @@
     "tldr-lint": "^0.0.13"
   },
   "devDependencies": {
-    "husky": "^7.0.4"
+    "husky": "^7.0.4",
+    "markdownlint": "^0.26.2"
```

Comment:
Why you added this? `markdownlint` is a direct production dependency from `markdownlint-cli`. This is not required.